### ISSUE TITLE
importing polymer-redux with meteor

### ISSUE THREAD

**Issue by yorrd** [tur-nr/polymer-redux#111]
I'm having a problem with importing polymer-redux in a meteor/polymer project.
It's saying that PolymerRedux is not defined but I'm pretty sure I am importing it correctly.
I solved the problem by removing the var keyword in front of PolymerRedux in dist/polymer-redux.html. But I don't know if that's a nice way to do this.

**Comment by tur-nr** [comment]
Are you bundling? Removing the `var` will make it global when bundling so that's why it works.

**Comment by yorrd** [comment]
@tur-nr yes, Meteor is bundling. How would you go about solving this properly in your opinion? :)
